site/publish.sh

```diff
@@ -131,52 +131,51 @@ then
     export -f parallelfunction
 
     ############# Prepare the different UC source directories to be copied to different destinations
-    chmod -R a+r "${www2_dir}"
-    date +%s > "${www2_dir}"/TIME # TIME sync, used by mirrorbits to know the last update date to take in account
+    chmod -R a+r "${www2_dir}" # Required for updates.jenkins.io rsync copy using distinct user than httpd's
+    date +%s > "${www2_dir}"/TIME # Used by mirrorbits and healthchecks
 
-    # Note: these PATH must map to the FILESHARE_SYNC_SOURCE in the ZIP env files (!). Yeah not easy but that's how it works temporarily.
+    # Note: these PATH must map to the FILESHARE_SYNC_SOURCE in the ZIP env files (!)
     httpd_secured_dir=./www-redirections-secured
     httpd_unsecured_dir=./www-redirections-unsecured
+    content_dir=./www-content
 
-    ## No need to remove the symlinks as the `azcopy sync` for symlinks is not yet supported and we use `--no-follow-symlinks` for `aws s3 sync`
-    # Perform a copy with dereference symlink (object storage do not support symlinks)
-    rm -rf ./www-content/ "${httpd_secured_dir}" "${httpd_unsecured_dir}" # Cleanup
+    # Cleanup
+    rm -rf "${content_dir}" "${httpd_secured_dir}" "${httpd_unsecured_dir}"
 
-    # Prepare www-content, a copy of the $www2_dir dedicated to mirrorbits service, excluding every .htaccess files
+    # Prepare www-content dir destined to the mirrorbits service
+    # By retrieving all JSON files from $www2_dir
+    # and dereferencing all internal symlinks to files (as AWS and azcopy CLIs do not support symlinks)
+    # NOTE: order of include and exclude flags MATTERS A LOT
     rsync --archive --verbose \
-        --copy-links `# derefence symlinks` \
+        --copy-links `# derefence symlinks to avoid issues with aws s3 or azcopy` \
         --safe-links `# ignore symlinks outside of copied tree` \
         --prune-empty-dirs `# Do not copy empty directories` \
         --exclude='updates/' `# Exclude ALL 'updates' directories, not only the root /updates (because symlink dereferencing create additional directories` \
-        --exclude='.htaccess' `# Exclude every .htaccess files` \
-        --exclude="uctest.json" `# Service Applicative Healthcheck (empty JSON)` \
+        --exclude="uctest.json" `# Service Applicative Healthcheck (empty JSON) only used by Apache` \
```

Comment:
not sure what the applicative is meaning here
```suggestion
        --exclude="uctest.json" `# Service Healthcheck (empty JSON) only used by Apache` \
```